src/wp-admin/css/site-icon.css

```diff
@@ -70,3 +69,10 @@
 	border-color: #b32d2e;
 	box-shadow: 0 0 0 1px #b32d2e;
 }
+
+.site-icon-section .action-buttons {
+	display: flex;
+	flex-wrap: wrap;
+	gap: 10px;
+}
+
```

Comment:
Empty line should be removed.

Reply:
Thanks for spotting this, [adf2331](https://github.com/WordPress/wordpress-develop/pull/6101/commits/adf2331ec1c396238cc0cf1550286c85378abe36) addresses this.